guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -150,20 +215,26 @@ Generic image:: A boot disk for PXE-less provisioning that is not tied to a spec
 The generic image sends the host’s MAC address to {ProjectServer}, which matches it against the host entry.
 
 [[Hammer]]
-Hammer:: A command line tool for managing {ProjectName}.
-You can execute Hammer commands from the command line or utilize them in scripts.
-Hammer also provides an interactive shell.
+Hammer::
+Hammer is a command-line interface tool for {Project}.
+You can execute Hammer commands from the command line or utilize it in scripts.
+You can use Hammer to automate certain recurring tasks as an alternative to {Project} Ansible collection or {Project} API.
 
 [[Host]]
-Host:: Refers to any system, either physical or virtual, that {ProjectName} manages.
+Host::
+A host is a physical, virtual, or cloud instance registered to {Project}.
 
 [[Host_collection]]
 Host collection:: A user defined group of one or more Hosts used for bulk actions such as errata installation.
 
 [[Host_group]]
-Host group:: A template for building a host.
-Host groups hold shared parameters, such as subnet or lifecycle environment, that are inherited by host group members.
-Host groups can be nested to create a hierarchical structure.
+Host group::
+A host group is a template to build hosts that holds shared parameters, such as subnet or lifecycle environment.
+It helps to unify configuration management in Ansible, Puppet, and Salt by grouping hosts.
+You can nest host groups to create a hierarchical structure.
+// ACK?
+Host groups apply not only to new hosts, but also existing ones.
+Note that this excludes the content view and lifecycle environment, which is defined by the activation key.
```

Comment:
Hosts inherit attributes from their assigned host group, but only if the host does not already have a value for the attribute. As of https://github.com/Katello/katello/pull/10841, content view and lifecycle environment do not behave any differently than other fields.

For existing hosts, Puppet class settings are the only ones that are "sticky" - https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#Working_with_Host_Groups_managing-hosts

Reply:
Thanks, reworded.